### DISCUSSION TITLE
Compact mode

### DISCUSSION THREAD

**Discussion by Drahsid** [stoatchat/discussions#549] [Feature Suggestions]
I looked through the other discussions and did not see a similar discussion, so forgive me if I missed a duplicate.
Other chat clients that I have used either never had, or had an option to omit profile pictures in chats, for a more compact viewing. I prefer this, since to me, profile pictures convey no useful information, and I would rather see more of a discussion. I think this would improve Revolt for some people who have similar preferences.

**Comment by gatomod** [comment]
Not a bad idea. I think you should request it as a feature at new client repo
